src/components/TaskItem/TaskItem.tsx

```diff
@@ -33,30 +23,31 @@ const TaskItem = ({
 
   return (
     <div className={styles.TaskItemComponent}>
-      <Checkbox checked={checked} onChange={() => setChecked(!checked)} />
+      <Checkbox checked={taskChecked} onChange={() => setChecked(!taskChecked)} />
       <input
         ref={ref}
+        id={id}
         className={clsx(styles.taskInput, {
-          [styles.taskInputEditable]: isEditable && !checked,
-          [styles.taskInputCompleted]: checked,
+          [styles.taskInputEditable]: isEditable && !taskChecked,
+          [styles.taskInputCompleted]: taskChecked,
```

Comment:
why you wouldn't name the classess similar to internal state - taskEditable, taskCompleted/Checked?